pytorch_blade/src/compiler/jit/tool_funcs.cpp

```diff
@@ -138,6 +138,16 @@ torch::TypePtr fromNumberType(torch::TypePtr typ) {
   return nullptr;
 }
 
+bool cast_to_i32_tensor_type(torch::jit::Value& value) {
```

Comment:
Looks to me here just simply change the input's scalar type at build time? But I think at runtime user will still pass long values to the trt engine, wouldn't that cause type mismatch?

Reply:
In [`TRTContext::PreProcessInputs`](https://github.com/fortianyou/BladeDISC/blob/features/fix_trt/pytorch_blade/src/compiler/tensorrt/tensorrt_engine_context.cpp#L334)  will convert the input type to the corresponding graph input type at runtime. And the graph input type is recorded at build time.

Reply:
I see